modules/TrackPileUpSubtractor.cc

```diff
@@ -116,23 +116,45 @@ void TrackPileUpSubtractor::Finish()
 
 void TrackPileUpSubtractor::Process()
```

Comment:
I think we want to have a new module instead of replacing the existing one because there are workflows that depend on it.